### ISSUE TITLE
no-duplicates does not add whitespace after commas

### ISSUE THREAD

**Issue by resolritter** [benmosher/eslint-plugin-import#1956]
I had some code like this

```js
import { a } from "pkg"
import { b } from "pkg"
```

"import/no-duplicates" did work, but it left `a` and `b` glued together like so

```js
import { a,b } from "pkg"
```

Having the imports glued by commas upsets Prettier. A sensible solution is to add a whitespace after the comma(s).

```js
import { a, b } from "pkg"
```

**Comment by resolritter** [comment]
It seems that setting up [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) gets around this... But it'd be nice if it could add the whitespaces by itself

**Comment by ljharb** [comment]
prettier should only ever be run via eslint anyways :-)

an alternative solution is to use the comma-spacing eslint rule to correct it. I’d be fine changing this rule to add the whitespaces by default tho, since that’s by far the more common style.
